src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -789,7 +789,7 @@
           },
           {
             "condition": "(tfm == 'net8.0')",
-            "value": "8.0.0-rc.1.9171"
+            "value": "8.0.0-rc.2.9373"
```

Comment:
This will probably break the build - at a guess we'll need to update the net8 sdk and rollback file for workloads